src/nodes/uvc_ros_driver_node.cpp

```diff
@@ -85,11 +94,18 @@ int main(int argc, char **argv)
 	nh.getParam("depthMap", depth_map);
 	nh.getParam("cameraConfigFile", calibration_file_path);
 	nh.getParam("calibrationMode", calibration_mode);
+	nh.getParam("primaryCamMode", primary_camera_mode);
+
+	//force cam number to be even
+	if (number_of_cameras % 2 != 0){
```

Comment:
is this necessary? what about having a fatal error instead?

Reply:
Even if you request an odd number of cameras it gives you an even number in the callback, I just did this so I don't have to check before publishing. Though it could be done in a cleaner way